src/main/java/nl/praegus/fitnesse/symbols/MavenProjectVersions/VersionNumber.java

```diff
@@ -0,0 +1,32 @@
+package nl.praegus.fitnesse.symbols.MavenProjectVersions;
+
+public class VersionNumber{
+    public final int[] versionNumberArray;
+
+    public VersionNumber(String version) {
```

Comment:
Deze klasse kan ook wel een paar unittests gebruiken. Er zit hier nogal wat logica in!

Reply:
Gemaakte unit tests in een andere test klasse geplaatst.